Automattic/jetpack
# Jetpack Settings: Add new entry link for AI assistant features

Fixes https://github.com/Automattic/jetpack-roadmap/issues/1721

## Proposed changes:
Add an entry on settings -> write -> composing to link to AI Assistant product features page.
<img width="503" alt="image" src="https://github.com/user-attachments/assets/ecb68daa-c8b9-4b2c-accd-16bc31801b9c">

### Other information:


## Jetpack product discussion
p1HpG7-t9s-p2

## Does this pull request change what data or activity we track or use?
No

## Testing instructions:
Spin up a JN site. Go to Jetpack -> Settings -> Writing. Last entry should be a link to My Jetpack's product page for Ai Assistant.

The link depends on the JP Blocks being enabled, try disabling JP Blocks. The link should become disabled.
